src/Pirouette/Term/Symbolic/Prover/Runner.hs

```diff
@@ -23,25 +24,33 @@ data AssumeProve lang = Term lang :==>: Term lang
 type IncorrectnessResult lang =
   Either String (Maybe (Path lang (EvaluationWitness lang)))
 
+-- For now only out of fuel stopping condition
+newtype StoppingCondition = StopOutOfFuel Int
+
+-- | Parameters for an incorrectness logic run
+data ILParams lang = ILParams
```

Comment:
To remain consistent with the other functions, maybe we should call this type `IncorrectnessLogicParams`; I'm fine with using the `il` prefix for the constructor and its fields.